.github/workflows/container-image.yml

```diff
@@ -90,13 +90,14 @@ jobs:
           make download-spdx-sbom-generator sbom
```

Comment:
I think this now could be substituted by the new github action, [bom-generator-installer](https://github.com/kubewarden/github-actions/blob/0e8fe58ef0bdcba71e355dfa4bb33c5ae54e7ec4/sbom-generator-installer/action.yaml).